state/runtime/addresslist/addresslist.go

```diff
@@ -146,6 +195,17 @@ func (a *AddressList) GetRole(addr types.Address) Role {
 	return Role(res)
 }
 
+func (a *AddressList) IsEnabled() bool {
+	return a.state.GetStorage(a.addr, enabledKeyHash) != types.ZeroHash
+}
+
+func (a *AddressList) SetEnabled(value bool) {
+	vl, _ := abi.MustNewType("bool").Encode(value)
```

Comment:
Extract `abiBoolTrue` and `abiBoolFalse` to variables and use it as necessary (same thing was done in `precompiled.go`, so maybe we can extract those variables to some common package and use it from there).

Reply:
This is only used for enabling/disabling list which will not be done frequently.
I will just make an internal func
```
func getAbiBoolValue(value bool) []byte {
	encodedValue, _ := abi.MustNewType("bool").Encode(value)

	return encodedValue
}
```
inside `state/runtime/addresslist/addresslist.go`. I am not sure where should I put that generic function, we can change that later